provider/cluster_rosa_classic_resource.go

```diff
@@ -1428,25 +1374,6 @@ func populateRosaClassicClusterState(ctx context.Context, object *cmv1.Cluster,
 	state.Domain = types.String{
 		Value: fmt.Sprintf("%s.%s", object.Name(), object.DNS().BaseDomain()),
 	}
-	state.Replicas = types.Int64{
```

Comment:
I think it still need to be computed - if someone will edit it from rosa cli we would like to be aligned with that changes